sdk/purview/azure-purview-sharing/azure/purview/sharing/aio/operations/_operations.py

```diff
@@ -1205,22 +1206,14 @@ def get_long_running_output(pipeline_response):
 
     @distributed_trace
     def list_attached(
-        self,
-        *,
-        reference_name: str,
-        skip_token: Optional[str] = None,
-        filter: Optional[str] = None,
-        orderby: Optional[str] = None,
```

Comment:
Shouldn't it `order_by`?

Reply:
All changes to this file involved what the Open API Specs team recommended

Reply:
There was more discussion surrounding this parameter in the open api spec review: https://github.com/Azure/azure-rest-api-specs/pull/24109#pullrequestreview-1439756253.  Ultimately, it sounds like the conventions of the language should be applied.  We can add a transform.